#!/usr/bin/env python
PACKAGE = "deep_nav_layers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("seg_height", int_t, 0, "Height of segmentation network output", 480, 1, 10000)
gen.add("seg_width", int_t, 0, "Width of segmentation network output", 960, 1, 10000)
gen.add("costmap_height", int_t, 0, "Height of the image representing a top-down view of the real world", 480, 1, 10000)
gen.add("costmap_width", int_t, 0, "Width of the image representing a top-down view of the real world", 960, 1, 10000)
gen.add("m_per_pixel", double_t, 0, "Meters per pixel", 0.05, 0, 1)
gen.add("min_x", int_t, 0, "Min X", -10, -50, 50)
gen.add("min_y", int_t, 0, "Min Y", -10, -50, 50)
gen.add("max_x", int_t, 0, "Max X", 10, -50, 50)
gen.add("max_y", int_t, 0, "Max Y", 10, -50, 50)
gen.add("enabled", bool_t, 0, "Enabled", True)

exit(gen.generate(PACKAGE, "deep_nav_layers", "deep_nav_layers"))
